pyproject.toml

```diff
@@ -22,18 +22,19 @@ cherrypy = "^18.2"
 bugsnag = "^3.6"
 requests = "^2.22"
 slackclient = "^1"
+python-dotenv = "^0.10.5"
```

Comment:
I'm a bit apprehensive to add in a dependency like this. Have you tried something like `direnv` instead?

Reply:
I have not, I believe dotenv was suggested by @itsthejoker so he may have some insight on this.

Reply:
direnv requires installing an extra piece of software on developer's machines and requires leaving a directory and re-entering it to refresh variables. dotenv is built into the application and only requires restarting the application to update variables. Given the choice, I'd much rather go with dotenv.

Reply:
Ah, we can take the argument of dotenv vs. direnv elsewhere, since either one will accomplish what we want. We'll discuss removal (if that's the direction we go) as a separate topic outside of this PR. :+1: